docs/4-the-sound-check/1-enable-monitoring.md

```diff
@@ -0,0 +1,64 @@
+## User Workload Monitoring
+
+> OpenShift’s has monitoring capabilities built in. It deploys the Prometheus stack and integrates into the OpenShift UI for consuming cluster metrics. 
+
+1. If click on `jukebox` model, you can see the graphs about incoming requests, CPU and memory consumption to give you an idea about the model utilization. 
+
+#### TODO: insert graph here
+
+2. There are other metrics OpenShift collects out of the box. You can run queries for the existing metrics easily with `promql`, a query language for Prometheus. On the OpenShift UI, go to Observe, it should show basic health indicators just like OpenShift AI UI.
+
+
+3. Run a promql query to get some info about the successfull requests to `jukebox` in your test environment.
+
+    ```bash
+    ovms_requests_success{namespace="<USER_NAME>-test", interface="REST"}
+    ```
+
+_You can generate some traffic by running some querries from the Jukebox UI 🎶_
+
+
+### Deploy Grafana
+
+> Let's create some more dashboards with specific information about our model!
+
+1. Let's deploy a Grafana instance in our `mlops` environment. Yet another tooling to support the end to end journey of Jukebox! Therefore we need to install it through `mlops-gitops/toolings/values.yaml`
+
+    ```bash
+      # Grafana
+      - name: grafana
+        enabled: true
+        source: https://gitea-gitea.apps.cluster-75vtz.75vtz.sandbox1169.opentlc.com/user1/mlops-helmcharts.git
```

Comment:
we want to anonymization for this?

Reply:
ah damn, I was copy pasting my tests of course :D haha thanks!